run_service.sh

```diff
@@ -1055,7 +1055,6 @@ export ON_CHAIN_SERVICE_ID=$service_id
 export ALL_PARTICIPANTS='["'$agent_address'"]'
 # This is the default market creator. Feel free to update with other market creators
 export OMEN_CREATORS='["0x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec"]'
-export BET_THRESHOLD=5000000000000000
```

Comment:
Using the default one, which is set to 10 cents.

Reply:
Even if this is the case, wouldn't it be ok to set it explicitly here? (so that users don't have to dig into the yaml to find the variable)